examples/angular_bazel_architect/package.json

```diff
@@ -9,9 +9,12 @@
     "lint": "bazel run :lint",
     "e2e": "bazel test :e2e",
     "postinstall": "patch-package && yarn webdriver-update && ngcc --properties es2015 browser module main --first-only",
-    "webdriver-update": "webdriver-manager update --standalone false --gecko false --versions.chrome 85.0.4183.38"
+    "webdriver-update": "webdriver-manager update --standalone false --gecko false --versions.chrome 101.0.4951.41"
   },
   "private": true,
+  "resolutions": {
+    "**/webdriver-manager": "12.1.8"
```

Comment:
Why was this one required?

Reply:
`12.1.8` has a fix for downloading chrome drivers on m1s. Protractor is no longer maintained and has a dependency on `12.1.7`. Is there another way I can get the lock file to resolve to `12.1.8`?

Reply:
I think `yarn upgrade protractor` will update its dependencies.

This might still be worth keeping just to enforce it though? Can you do `^12.1.8` in `resolutions` since you really just want to enforce a minimum and not an exact versions?